### PR TITLE
add log artifacts in aks pipeline

### PR DESCRIPTION
**Pull request by monianshouhou**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
add log artifacts in ado for aks pipeline

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: .pipelines/scripts/run-e2e.sh [hunk 1]
```diff
@@ -31,8 +31,35 @@ get_random_location() {
...
+  ALL_PODS=($(kubectl get pod -A -owide | grep "kube-system" | awk '{printf("%s ",$2)}'))
+  for POD in "${ALL_PODS[@]}"; do
+      kubectl logs ${POD} -n kube-system > "${ARTIFACT_DIR}/kube-system/${POD}.log"
+  done
+  ALL_NODES=($(kubectl get node | grep "aks-" | awk '{printf("%s ",$1)}'))
+  LOG_IMAGE="mcr.microsoft.com/dotnet/runtime-deps:6.0"
+  for NODE in "${ALL_NODES[@]}"; do
+      NODE_LOG_DIR="${ARTIFACT_DIR}/node-log/${NODE}"
+      mkdir "${NODE_LOG_DIR}"
+      kubectl debug node/${NODE} -it --image=${LOG_IMAGE} -- /bin/sh -c "chroot /host journalctl --no-pager --output=short-precise" > "${NODE_LOG_DIR}/journal.log"
```
Comment (andyzhangx) [inline .pipelines/scripts/run-e2e.sh:46]: is it possible just using one script to collect all those logs?

Comment (monianshouhou) [reply .pipelines/scripts/run-e2e.sh]: > is it possible just using one script to collect all those logs?

Updated.
I don't find some convenient ways to compact the "kubectl debug" instructions. But I try to use arrays to make the code more concise. PTAL.

Comment (andyzhangx) [reply .pipelines/scripts/run-e2e.sh]: thanks, I think that's more clear.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

File: .pipelines/scripts/run-e2e.sh (hunk 1 above)
Comment (monianshouhou) [reply .pipelines/scripts/run-e2e.sh]: > thanks, I think that's more clear.

Thanks a lot for review!

**Pull request merged**